### PR TITLE
Add Pointnetwork.io

### PR DESCRIPTION
**Pull request by yurivin**

### CODE REVIEW

File: point/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      ],
+      "base": "apoint",
+      "name": "Point",
+      "display": "point",
+      "symbol": "POINT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/pointnetwork/pointnetwork-dashboard/f143abd8e76a652bca407457d179065381274930/assets/point-logo.svg",
+	"png": "https://raw.githubusercontent.com/pointnetwork/pointnetwork/develop/internal/explorer.point/src/assets/pointlogo.png"
+      },
+      "coingecko_id": "point"
```
Comment (sunnya97) [inline point/assetlist.json:25]: Do you already have this coingecko id?

Comment (JeremyParish69) [reply point/assetlist.json:25]: this cgid does not exist, and will cause the checks to fail.

File: point/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          "denom": "point",
+          "exponent": 18
+        }
+      ],
+      "base": "apoint",
+      "name": "Point",
+      "display": "point",
+      "symbol": "POINT",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/pointnetwork/pointnetwork-dashboard/f143abd8e76a652bca407457d179065381274930/assets/point-logo.svg",
```
Comment (JeremyParish69) [inline point/assetlist.json:22]: please use URIs that point to the images that are uploaded to the chain registry. (i.e., https://raw.githubusercontent.com/cosmos/chain-registry/master/point/images/point-logo.(png|svg)

Comment (DaMandal0rian) [reply point/assetlist.json:22]: @JeremyParish69 thanks will look into this

File: point/assetlist.json (hunk 1 above)
Comment (DaMandal0rian) [reply point/assetlist.json:25]: @sunnya97 not as yet, should we leave the coingecko id blank for now?

File: point/chain.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
+  "chain_id": "point_10687-1",
+  "bech32_prefix": "point",
```
Comment (JeremyParish69) [inline point/chain.json:8]: Still failing checks. Please register this bech32 prefix to SLIP173:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (yurivin) [reply point/chain.json:8]: Done. Please check.
https://github.com/satoshilabs/slips/pull/1404

Comment (JeremyParish69) [reply point/chain.json:8]: Thanks!

File: point/chain.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "point",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Point",
```
Comment (JeremyParish69) [inline point/chain.json:6]: Could this change to 'Point Network' please? so it will then match the SLIP173 registration

Comment (yurivin) [reply point/chain.json]: Done

**Pull request merged**